src/layouts/AssistenzaPage.jsx

```diff
@@ -0,0 +1,839 @@
+                    </label>
+                    <Select
+                      id="argument-select"
+                      inputId="argument-select-input"
+                      dataRefer="00N3N00000GCzFR"
+                      options={selectArgument}
+                      onChange={handleArgument}
+                      placeholder={selectPlaceholder}
+                      aria-label={selectPlaceholder}
+                      aria-describedby="mandatory-label"
+                      /* aria-invalid={errors.representative && 'true'}
+                    aria-labelledby={errors.representative && 'error-represent'} */
+                      className={`select`}
+                    />
+                    <select
+                      className="d-none"
+                      id="00N3N00000GCzFR"
+                      name="00N3N00000GCzFR"
+                      title="Argomento"
+                      required={true}
+                      onInvalid={customInvalid}
+                    >
+                      <option value="Accesso portale">Accesso portale</option>
+                      <option value="Iscrizione newsletter">Iscrizione newsletter</option>
+                      <option value="Misure">Misure</option>
+                      <option value="Avvisi/bandi">Avvisi/bandi</option>
+                      <option value="Enti beneficiari">Enti beneficiari</option>
+                      <option value="Spese ammissibili">Spese ammissibili</option>
+                      <option value="Generale">Generale</option>
+                    </select>
+                  </div>
+                </Col>
+                <Col xs={12} md={6} lg={4} className="offset-lg-1">
+                  <div className="select-wrapper">
+                    <label htmlFor="measure-select-input" className={classes.selectLabel}>
+                      {measureLabel}
+                    </label>
+                    <Select
+                      id="measure-select"
+                      inputId="measure-select-input"
+                      dataRefer="00N3N00000GCzFW"
+                      options={selectMeasure}
+                      onChange={handleMeasure}
+                      placeholder={selectPlaceholder}
+                      aria-label={selectPlaceholder}
+                      aria-describedby="mandatory-label"
+                      /* aria-invalid={errors.representative && 'true'}
+                    aria-labelledby={errors.representative && 'error-represent'} */
+                      className={`select`}
+                    />
+                    <select
+                      className="d-none"
+                      id="00N3N00000GCzFW"
+                      name="00N3N00000GCzFW"
+                      title="Misura"
+                      required={true}
+                      onInvalid={customInvalid}
+                    >
+                      <option value="1.2 Abilitazione e facilitazione migrazione al Cloud">
```

Comment:
some measures are missing here. Probably we should consider to load them from yaml file and then iterate here.